android/app/src/main/java/com/mattermost/helpers/DatabaseHelper.kt

```diff
@@ -250,6 +279,23 @@ class DatabaseHelper {
         }
     }
 
+    fun getServerVersion(db: Database): String? {
+        val config = getSystemConfig(db)
+        if (config != null) {
+            return config.getString("Version")
+        }
+        return null
+    }
+
+    private fun getSystemConfig(db: Database): JSONObject? {
+        val configRecord = find(db, "System", "config")
+        if (configRecord != null) {
+            val value = configRecord.getString("value");
+            return JSONObject(value)
```

Comment:
you should wrap this in a try/catch.. `getString` can return null then `JSONObject` will throw `NullPointerException`